### PR TITLE
Mocha install

### PR DESCRIPTION
**Pull request by Rainbacon**
I set up Mocha as a testing framework and added unit tests for the Skills service.

### CODE REVIEW

File: microservice/skill/index.js [hunk 1]
```diff
@@ -22,139 +25,19 @@ exports.handler = (event, context, callback) => {
...
-        console.log('Error: delete user ', event.queryStringParameters.id, ' in DynamoDB table', process.env.SKILL_TABLE);
-        console.log(err, err.stack);
-
-        const response = {
-            statusCode: 500,
-            body: JSON.stringify(err)
-        };
-        callback(err, null);
-    });
+    skill.update(event, callback);
```
Comment (paulSambolin) [inline microservice/skill/index.js]: should be delete??

Comment (Rainbacon) [inline microservice/skill/index.js]: Yes. I moved all of this code to another file.

File: package.json [hunk 2]
```diff
@@ -17,7 +17,11 @@
   },
   "homepage": "https://github.com/pariveda/sweet-skills-api#readme",
   "dependencies": {
+    "check-types": "^7.1.5",
+    "dotenv": "^4.0.0",
```
Comment (Rainbacon) [inline package.json]: I can change that

File: microservice/skill/index.js (hunk 1 above)
Comment (paulSambolin) [inline microservice/skill/index.js]: I mean if you scroll up you are calling update within the delete function and the update function

Comment (Rainbacon) [inline microservice/skill/index.js]: Oops, meant to have  that be the delete function

**Pull request merged**